testsuite/testsuite-jsp/pom.xml

```diff
@@ -49,6 +64,12 @@
             <version>${project.version}</version>
             <executions>
               <execution>
+                <id>package</id>
```

Comment:
Sorry, was getting confused thinking this was in -examples.

I'll ask a different question, why does the test need to be run as an uberjar? Does it not work without it?